### PR TITLE
feat: add support for Kubernetes v1.18.15 on Azure Stack

### PR DESCRIPTION
**Pull request by JesusAlvarezTorres**
**Reason for Change**:

Adding support for Kubernetes v1.18.15 on Azure Stack

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [X] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jackfrancis** [comment]
code lgtm

to @jadarsie for final review/merge

**Review by jadarsie — APPROVED** [review]

File: vhd/packer/install-dependencies.sh [hunk 1]
```diff
@@ -374,6 +374,7 @@ K8S_VERSIONS="
 1.19.7
 1.19.6
 1.18.15
+1.18.15-azs
 1.18.14
 1.18.10-azs
```
Comment (jadarsie) [inline vhd/packer/install-dependencies.sh:379]: should this line go away?

Comment (JesusAlvarezTorres) [reply vhd/packer/install-dependencies.sh:379]: dont you mean 1.18.10-azs? If so yes, missed it.

Comment (JesusAlvarezTorres) [reply vhd/packer/install-dependencies.sh]: removed  1.18.10-azs

**Comment by JesusAlvarezTorres** [comment]
Merged changes into https://github.com/Azure/aks-engine/pull/4187

**Pull request closed without merging**